pages/visit/locations/index.vue

```diff
@@ -76,74 +76,68 @@ const searchGenericQuery = ref({
       JSON.parse(route.query.filters)) ||
     {},
 })
-
-/* TODO: Refactor when search functionality is ready */
-/*
-  async fetch() {
-  // this.uclaLibraries = []
-  this.hits = []
+// ELASTICSEARCH
+async function searchES() {
+  hits.value = []
```

Comment:
there is some unwanted code, so replace this function with 
async function searchES() {
  if (
    (route.query.q && route.query.q !== '') ||
    (route.query.filters &&
      queryFilterHasValues(
        route.query.filters,
        config.locationsList.filters
      ))
  ) {
    console.log('Search ES HITS query,', route.query.q)
    const queryText = route.query.q || '*'
    const results = await $dataApi.keywordSearchWithFilters(
      queryText,
      config.locationsList.searchFields,
      "sectionHandle:location OR sectionHandle:affiliateLibrary'",
      (route.query.filters &&
        JSON.parse(route.query.filters)) ||
      {},
      config.locationsList.sortField,
      config.locationsList.orderBy,
      config.locationsList.resultFields,
      []
    )
    if (results && results.hits && results.hits.total.value > 0) {
      console.log('Search ES HITS,', results.hits.hits)
      hits.value = results.hits.hits
      noResultsFound.value = false
    } else {
      noResultsFound.value = true
      hits.value = []
    }
  } else {
    // console.log('data.value', data.value)
    hits.value = []
    noResultsFound.value = false
  }
}

Reply:
just made these changes ✅